Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOProjectCreator.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+import SwiftUI
+
+struct PPOProjectCreator: View {
+  let creatorName: String
+
+  var body: some View {
+    HStack(alignment: .firstTextBaseline) {
+      // TODO: Localize
+      Text("Created by **\(self.creatorName)**")
+        .font(Font(Constants.createdByFont))
+        .foregroundStyle(Color(Constants.createdByColor))
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .lineLimit(1)
+
+      // TODO: Localize
+      Text("Send a message")
+        .font(Font(Constants.sendMessageFont))
+        .foregroundStyle(Color(Constants.sendMessageColor))
+        .frame(alignment: .trailing)
+        .lineLimit(1)
+
+      Spacer()
+        .frame(width: 6)
```

Comment:
Nit: use `Styles.grid(1)` instead of hardcoding the `6`. (This is defined in This is defined in our `BaseStyles.swift` file with grid spacing of `6` - we also have `halfGrid` increments of `3`).